### ISSUE TITLE
Tx losses file has value larger than flows.csv

### ISSUE THREAD

**Issue by gschivley** [GenXProject/GenX#616]
The transmission losses reported from my model are a couple orders of magnitude larger than the absolute flows for each line. It looks like [this commit](https://github.com/GenXProject/GenX/commit/007c7271c54e3e8e445d65f20fd8b57d82d61016) changed the output calculation from multiplying by `ModelScalingFactor` to multiplying by `ModelScalingFactor^2`. 

@cfe316, am I reading the code correctly?

**Comment by cfe316** [comment]
While I believe your statement that there are anomalously high losses, the output even before that commit multiplied by ModelScalingFactor twice!

 ```
 for i in 1:L
	if i in inputs["LOSS_LINES"]
		tlosses[i,:] = value.(EP[:vTLOSS])[i,:] * ModelScalingFactor     # once
	end
	...
end
dfTLosses = hcat(dfTLosses, DataFrame(tlosses * ModelScalingFactor, :auto))     # twice
```

**Comment by lbonaldo** [comment]
Thank you, @gschivley, for opening this issue! Fixed in #622 (main) and #621 (develop).
